app/src/controllers/pages/actionCreators.js

```diff
@@ -14,9 +14,14 @@
  * limitations under the License.
  */
 
-import { UPDATE_PAGE_PROPERTIES } from './constants';
+import { UPDATE_PAGE_PROPERTIES, CLEAR_PAGE_STATE } from './constants';
 
 export const updatePagePropertiesAction = (properties) => ({
   type: UPDATE_PAGE_PROPERTIES,
   payload: properties,
 });
+
+export const clearPageStateAction = (oldPage) => ({
+  type: CLEAR_PAGE_STATE,
+  payload: { oldPage },
```

Comment:
Maybe just `payload: oldPage`? I think it's still clear what payload contains, no need to wrap it in an object.

Reply:
Fixed